### ISSUE TITLE
Login Pass streams

### ISSUE THREAD

**Issue by verema** [xteve-project/xTeVe#334]
Hello everyone, I have been using xteve as an iptv server for a long time, and I really love it, I have a doubt since I have realized that even if you create users with their pass, when you watch the stream you all have the same url, that is to say that it does not differ from User1 to User 2 example

User1-Pass
http://localhost:34400/stream/84da41fde4417e2bd5b0690c48e94fc8

User2-Pass
http://localhost:34400/stream/84da41fde4417e2bd5b0690c48e94fc8

Would there be a way to configure each user to stream with their user and pass from the channels not from the list, since anyone who has the list.m3u has access to the entire server without user and pass

User1-Pass
http://User1-Pass@localhost:34400/stream/84da41fde4417e2bd5b0690c48e94fc8

Since when you download the list.m3 from the server, the user and pass that you put do not matter, they are all the same example

http://localhost:34400/m3u/xteve.m3u?username=User1&password=Pass
xteve.m3u=

#EXTM3U url-tvg="http://localhost:34400/xmltv/xteve.xml" x-tvg-url="http://localhost:34400/xmltv/xteve.xml"
#EXTINF:0 channelID="x-ID.0" tvg-chno="1" tvg-name="" tvg-id="1" tvg-logo="" group-title="",?
http://localhost:34400/stream/84da41fde4417e2bd5b0690c48e94fc8
#EXTINF:0 channelID="x-ID.1" tvg-chno="2" tvg-name="" tvg-id="2" tvg-logo="" group-title="",??
http://localhost:34400/stream/04f0d47000d17e1b6a3e53a96b9b0a3b
#EXTINF:0 channelID="x-ID.2" tvg-chno="3" tvg-name="" tvg-id="3" tvg-logo="" group-title="",???
http://localhost:34400/stream/01282cf8be085644eb530cfb2fb4586c
#EXTINF:0 channelID="x-ID.3" tvg-chno="4" tvg-name="" tvg-id="4" tvg-logo="" group-title="",????
http://localhost:34400/stream/6214bc858b5135082c16bc23f53e4724
#EXTINF:0 channelID="x-ID.................

http://localhost:34400/m3u/xteve.m3u?username=User2&password=Pass
xteve.m3u =

#EXTM3U url-tvg="http://localhost:34400/xmltv/xteve.xml" x-tvg-url="http://localhost:34400/xmltv/xteve.xml"
#EXTINF:0 channelID="x-ID.0" tvg-chno="1" tvg-name="" tvg-id="1" tvg-logo="" group-title="",?
http://localhost:34400/stream/84da41fde4417e2bd5b0690c48e94fc8
#EXTINF:0 channelID="x-ID.1" tvg-chno="2" tvg-name="" tvg-id="2" tvg-logo="" group-title="",??
http://localhost:34400/stream/04f0d47000d17e1b6a3e53a96b9b0a3b
#EXTINF:0 channelID="x-ID.2" tvg-chno="3" tvg-name="" tvg-id="3" tvg-logo="" group-title="",???
http://localhost:34400/stream/01282cf8be085644eb530cfb2fb4586c
#EXTINF:0 channelID="x-ID.3" tvg-chno="4" tvg-name="" tvg-id="4" tvg-logo="" group-title="",????
http://localhost:34400/stream/6214bc858b5135082c16bc23f53e4724
#EXTINF:0 channelID="x-ID.................

any solution to this thanks

**Comment by MCTyler** [comment]
No. xTeVe is just a Proxy and doesn't have this capability. Be advised, the "user" can only access the stream if they have access to urls.json file. xTeVe is designed to be used with a Media Player like Plex or Emby where individuals can be controlled.

**Comment by verema** [comment]
thanks
